src/_layouts/base.html

```diff
@@ -58,6 +58,7 @@
     <link href="https://fonts.googleapis.com/css2?family=Google+Sans+Text:wght@400;500;700&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css2?family=Google+Sans+Mono:wght@400;500;700&display=swap" rel="stylesheet">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200" rel="stylesheet" />
```

Comment:
Is there a reason you have to use Material Symbols instead of Icons? I don't really recommend it until we migrate everything over since that will increase site load time on every page not insignificantly. The file this loads is almost 3MB currently, which is likely bigger than everything else combined.

If you can't use Material Icons, consider adding Symbols for just this page and use a reduced set with just the variants you need specified in the URL.

Reply:
It was an experiment. Let me see what I can do.

Reply:
Sounds good, thanks! Let me know if you have any issues :)

Reply:
Changed from dynamic to static. Now it's 239Kb

![image](https://github.com/flutter/website/assets/706219/851666ef-8a97-44b0-969f-cb86823b242e)

Reply:
Still quite large but since we'll move everything else soon enough, that seems fine to me. Thanks!